05.ls/ls.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+MAX_COLUMN = 3
+SPACE_LENGTH = 3
+
+def main
+  files = search_files
+  row_count = (files.size.to_f / MAX_COLUMN).ceil
+  display_width = ajust_width(files).max + SPACE_LENGTH
+  output(row_count, files, display_width)
```

Comment:
```suggestion
  output(files)
```

row_countもdisplay_widthも、filesがあれば導出可能なので、引数は一個だけで済むかなーと思いました。

Reply:
言われてみればその通りでした。両変数はoutputメソッドに移行させました。